src/main/java/core/basesyntax/strategy/IncrementHandler.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.service.StorageService;
+
+public class IncrementHandler implements OperationHandler {
+    private final StorageService storageService;
+
+    public IncrementHandler(StorageService storageService) {
+        this.storageService = storageService;
+    }
+
+    @Override
+    public void handle(String fruitName, int quantity) {
+        Fruit fruit = new Fruit(fruitName);
+        storageService.increment(fruit, quantity);
+    }
```

Comment:
According to the checklist, classes should not start with an empty line. Please remove the empty line at the beginning of the class definition.